### ISSUE TITLE
Add support for xUnit Traits in tests

### ISSUE THREAD

**Issue by Eilon** [aspnet/Tooling#182]
From https://github.com/aspnet/Testing/issues/136

We need a mechanism to pass through the data for xUnit's Traits and similar types of data from the test framework.

https://github.com/xunit/xunit/issues/487#issuecomment-127652743

**Comment by sayedihashimi** [comment]
TFS: 149249
Added it to the backlog, will get back here once we have more.

**Comment by herecydev** [comment]
Any update on this?

**Comment by shederman** [comment]
This has had quite a runaround, from https://github.com/xunit/xunit/issues/485, to https://github.com/aspnet/Testing/issues/136, and now here.

Is this where it's going to get fixed, or is the potato just going to get passed on?

Because it's really quite an annoying bug, have all these lovely traits for my tests, and they're just ignored.

**Comment by arashnorouzi** [comment]
I was wondering if there has been any updates on this. 
Thanks.

**Comment by DerAlbertCom** [comment]
yep, currently i'm writing the the MSpec Test Adapter for DNX, and was searching for grouping the testing VS by traits. Because i would like to but the MSpec Subject there. After trying some hours i found this. So, please ;)

**Comment by sayedihashimi** [comment]
We are in the process of converting vs unit testing to be based on dotnet instead of dnx. Because of this we have not added any features to bs unit testing yet. After RC2 we should be able to add features like this.

**Comment by GunterGlad** [comment]
RC2 is out now and any update? Please?

**Comment by GmsDave** [comment]
Have Preview 2 of Tooling - still not working? Looking doubtful this will make it in for Tooling RTM.

**Comment by another-guy** [comment]
What is the status of this issue?

**Comment by dmccaffery** [comment]
@sayedihashimi this is killing me now that the only way to filter tests through dotnet test (msbuild) needs this info too!

**Comment by another-guy** [comment]
Any update? Is there any way to vote on this issue to make it a high priority???

As of v `14.0.25431.01 Update 3` VS seems to not treat the Traits... Running tests not based on traits is very annoying.
